webpack/assets/javascripts/react_app/components/common/DocumentationLink/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { MenuItem } from 'patternfly-react';
+import Icon from '../Icon';
+
+function onClickedLink(e) {
+  e.preventDefault();
+  window.open(this, '_blank');
+}
+
+export default ({ href, id = 'documentationLink' }) => (
+  <MenuItem key="documentationUrl" id={id} href={href} onClick={onClickedLink.bind(href)}>
```

Comment:
tbh, I prefer the `onClickedLink` inside the component and without `bind`, without `this` and with ES2015 syntax.

Reply:
mind providing an example?

Reply:
```js
import React from 'react';
import { MenuItem } from 'patternfly-react';
import Icon from '../Icon';

export default ({ href, id = 'documentationLink' }) => {
  const handleClick = (e) => {
    e.preventDefault();
    window.open(href, '_blank');
  };

  return (
    <MenuItem key="documentationUrl" id={id} href={href} onClick={handleClick}>
      <Icon type="question-sign" className="icon-black" />
      {` ${__('Documentation')}`}
    </MenuItem>
  );
};
```